admin_guide/install/advanced_install.adoc

```diff
@@ -197,6 +401,101 @@ node2.example.com         kubernetes.io/hostname=node2.example.com,region=primar
 ----
 ====
 
+*Multiple etcd Hosts*
+
+If you installed multiple *etcd* hosts:
+
+. Verify the *etcd* cluster health, substituting for your FQDNs and number of
+hosts in the following:
++
+====
+----
+# etcdctl -C \
+    https://etcd1.example.com:2379,https://etcd2.example.com:2379,https://etcd3.example.com:2379 \
+    --ca-file /etc/etcd/ca.crt \
+    --cert-file=/etc/openshift/master/master.etcd-client.crt \
+    --key-file=/etc/openshift/master/master.etcd-client.key cluster-health
```

Comment:
We should specify that this command is expected to be run on the master (run on an etcd host or a node host and the referenced cert-file and key-file will not exist.

Also, we should probably use the etcd ca cert that is in /etc/openshift/master (master.etcd-ca.crt) as well, since the command referenced above would *only* work on a host that is both an etcd host and a master host.